sentences/de/climate_HassClimateGetTemperature.yaml

```
@@ -7,4 +7,6 @@ intents:
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
           - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie warm ist es <area>"
+          - "Wie heiß ist es <area>"
+          - "Wie kalt ist es <area>"
```

these 3 can be merged into one

```suggestion
          - "Wie (warm | heiß | kalt) ist es <area>"
```

---

that makes sense, thanks